src/main/java/spoon/reflect/code/CtStatementList.java

```diff
@@ -48,6 +48,11 @@
 	<T extends CtStatementList> T addStatement(CtStatement statement);
 
 	/**
+	 * Inserts the given statement at a specific position in the list of statements
```

Comment:
Shifts the statement currently at that position (if any) and any subsequent statements to the right (adds one to their indices).